src/ui.rs

```diff
@@ -97,7 +93,7 @@ impl Terminal {
             execute!(stdout(), LeaveAlternateScreen, Show).unwrap();
             eprintln!("{}", e);
         }));
-        execute!(self.stdout, EnterAlternateScreen, Clear(ClType::All), DisableLineWrap)?;
+        execute!(self.stdout, EnterAlternateScreen, Clear(ClType::All), DisableLineWrap, EnableMouseCapture)?;
```

Comment:
Something to check: do we need to disable mouse capture after leaving the editor?